Add some missing environment variables. (#90)
kasenvr/vircadia-docs-sphinx: docs/source/developer/interface/environment-variables.md

# Environment Variables

You can alter Interface's functionality using the following environment variables.

---

Use the source files for QML and other resources at runtime instead of the compiled files.

    HIFI_USE_SOURCE_TREE_RESOURCES=1

---

> Enable the OpenVR plugin to be used with an Oculus HMD. (Useful for debugging the OpenVR plugin if you have an Oculus HMD but not a Vive.)

Reviewer: ```suggestion
Enable the OpenVR plugin to be used with an Oculus HMD. (Useful for debugging the OpenVR plugin if you have an Oculus HMD.)
```
There is many more SteamVR HMDs other than the Vive nowadays.

Reviewer 2: This is a developer debugging aid: the Oculus controllers don't work properly with this setting enabled, for example, It just forces the iopenvr.dll plugin to be used instead of oculus.dll plugin. ... So I'd say no text change needed.

Reviewer: Did you try that out? The Oculus controllers should work perfectly fine via OpenVR.

Reviewer 2: Yes, I don't have a Vive but have used this setting for debugging the Vive plugin. The Oculus controllers move your hands OK but the button and trigger mappings aren't handled.

> You can alter Interface's functionality using these environment variables.

Reviewer 2: ```suggestion
You can alter Interface's functionality using the following environment variables.
```

---

Use the source files for QML and other resources at runtime instead of the compiled files.

    HIFI_USE_SOURCE_TREE_RESOURCES=1

---

> Enables the OpenVR plugin to be used with an Oculus HMD. (Useful for debugging the OpenVR plugin if you have an Oculus HMD but not a Vive.)

Reviewer 2: ```suggestion
Enable the OpenVR plugin to be used with an Oculus HMD. (Useful for debugging the OpenVR plugin if you have an Oculus HMD but not a Vive.)
```

    HIFI_DEBUG_OPENVR=1

---

Override the metaverse URL.

    HIFI_METAVERSE_URL=<metaverse-server-url>

---

> Allow multiple instances of Interface to be running simultaneously.

Reviewer 2: ```suggestion
Allow multiple instances of Interface to run simultaneously.
```

    HIFI_ALLOW_MULTIPLE_INSTANCES=1

---

Logging to the console can be controlled with the `VIRCADIA_LOG_OPTIONS` variable. It can be set to any comma separated combination of the following options. Case and whitespace are not significant.

* `color` - Force ANSI colors depending on the message's severity.
* `nocolor` - Force ANSI colors off.
* `milliseconds` - Log timestamps with millisecond precision.
* `process_id` - Log the process ID.
* `thread_id` - Log the thread ID.

For example:

    VIRCADIA_LOG_OPTIONS="color, milliseconds"

> Will result in logging with millisecond precision, and color coding messages by severity.

Reviewer 2: ```suggestion
This will cause logging with millisecond precision, and color coding messages by severity.
```

> Color is enabled by default on Unix, if output is being directed to a terminal, and disabled by default on Windows.

Reviewer 2: ```suggestion
Color is enabled by default on Unix if output is being directed to a terminal, and disabled by default on Windows.
```
